### ISSUE TITLE
Create a responsibilities doc for CoP

### ISSUE THREAD

**Issue by ebele-oputa** [hackforla/product-management#308]
### Overview
We need to create a document that details the responsibilities of the product management community of practice lead/co-leads so that PMs know what it entails and can decide if they want to volunteer or not.

### Action Items
- [x] Research on similar docs
- [x] Draft doc
- [ ] Review doc with Bonnie, Carol and Priyanka

### Resources/Instructions
[PM CoP Lead Responsibilities Doc](https://docs.google.com/document/d/17fRfBF2beJprtBLZD_u4YVg6Nw5g0khvDCJJBJADZ38/edit)

**Comment by ebele-oputa** [comment]
@priyatalwar have you looked at the document?

**Comment by ebele-oputa** [comment]
review this info and determine if you want to add any of this to the responsibilities list:
a.     Onboarding Support for New Teammate’s by creating a holistic curriculum
b.     Rethink the leadership plan for PDM’s
c.     Development of a Hack for LA Product Roadmap
d.     Creating and socialize the Community of Practice for PDM’s
e.     Support the Career Development of PDM’s and provide a framework for scalability i.     Data Science skill set development
f.      Creation of Product Reporting Structure for Hack for LA
g.     Support the overall strategy and implementation of all initiatives for Hack for LA
h.     Stakeholder Happiness Officer (Just thought of this one)

**Comment by priyatalwar** [comment]
- Use the OM CoP to assign PMs to contribute to a PM guide
 - support projects in recruiting New PMs
 - make sure new PMs have attended onboarding, assigned to a project, have access to PM repository and have added themselves to the roster 
 - make sure meeting is being recorded and the recording link is uploaded to the appropriate folder
- facilitates at least 2 meetings a month with a prepared and communicated agenda prior to the meeting

**Comment by priyatalwar** [comment]
@carolzjy feel free to add anything to this doc that we may have missed so we can review and finalize at our meeting next week.

**Comment by ebele-oputa** [comment]
@priyatalwar added your list to the document

**Comment by ExperimentsInHonesty** [comment]
- [ ] reach out to Kevin Wange from UI/UX CoP he has created some documents for his CoP

**Comment by carolzjy** [comment]
Update: reached out to Kevin Wang, Greg Pawin, and QiQi on Slack on 11/17. No response yet. Will follow up again on 11/24

**Comment by ExperimentsInHonesty** [comment]
@ebele-oputa please review my comments in the document

**Comment by ExperimentsInHonesty** [comment]
@priyatalwar after Ebele makes her updates, lets reword this to be designed for enticement and direct resume application

**Comment by ebele-oputa** [comment]
Update:
- I've reviewed Bonnie's comments and made changes to the document
- I need HfLA's communication style guide for google docs to make the document compliant with org's policies
